tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/StaticMethods.java

```diff
@@ -645,8 +645,17 @@ public static Object getValue() {
         return ValueCreator.createMapValue(recordType, mapInitialValueEntries);
     }
 
-    public static BDecimal defaultDecimalArgs(BDecimal a, BDecimal b) {
+    public static BDecimal defaultDecimalArgsAddition(BDecimal a, BDecimal b) {
         BDecimal c = a.add(b);
         return c;
     }
+
+    public static Object defaultDecimalArgs(String s, BDecimal d) {
+        if (!d.booleanValue()) {
```

Comment:
Why need to check boolean here?

Reply:
As per a random test, I wanted to ensure that the `BDecimal` is passed correctly. Is it seems more irrelevant?

Reply:
But it is always true based on your test?